src/mixins/files.ts

```diff
@@ -150,6 +150,43 @@ export default class FilesMixin extends Vue {
       })
   }
 
+  enqueueFile (filename: string, path: string) {
+    path = path.replace(/gcodes\/?/gi, '')
+    const filepath = (path) ? `${path}/${filename}` : `${filename}`
+    const filenames = [filepath]
+    const data = {
+      filenames: filenames
+    }
+    return httpClient
+      .post(
+        this.apiUrl + '/server/job_queue/job',
+        data
+      )
+      .then((response) => {
+        this.updateQueue()
+        return response
+      })
+      .catch(e => {
+        return e
+      })
+  }
+
+  updateQueue () {
```

Comment:
Shouldn't the updates come in over the WS? Specifically `socket/notifyJobQueueChanged`

Reply:
Changed!